src/knl/knl_vadv_variant_ij_blocked_i_vector.h

```diff
@@ -0,0 +1,188 @@
+#pragma once
+
+#include "knl/knl_platform.h"
+#include "knl/knl_vadv_variant.h"
+#include "omp.h"
+
+#define eproma 64 
```

Comment:
Use constexpr constant instead of macro?

Reply:
I fixed it. Not entirely sure this is the correct way.